### PR TITLE
Trackers: typesquare.com

### PR DESCRIPTION
**Pull request by ghost**
### URL: `https://cdp-japan.jp/`

`typesquare.com` is a provider that provide a third-party font service.
https://typesquare.com

### Network Logs
network1

### CODE REVIEW

**Comment by piquark6046** [comment]
ryanbr/fanboy-adblock#1454

File: SpywareFilter/sections/specific.txt [hunk 1]
```diff
@@ -22,6 +22,8 @@
 ||nd.demdex.net/event?d_mid=$redirect=nooptext,important,domain=9now.com.au
 ||adc.nine.com.au/?token=$redirect=nooptext,important,domain=9now.com.au
 !
+||wf.typesquare.com^$~font
```
Comment (Alex-302) [inline SpywareFilter/sections/specific.txt:25]: Not sure about this rule. I see only fonts on wf.typesquare.com

File: SpywareFilter/sections/specific.txt [hunk 2]
```diff
@@ -22,6 +22,8 @@
...
+k*ck*ss*/static/cache/?url=https://mc.yandex.ru/
```
Comment (Alex-302) [inline SpywareFilter/sections/specific.txt:26]: wrong rule. Please remove.

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: SpywareFilter/sections/specific.txt (hunk 2 above)
Comment (piquark6046) [reply SpywareFilter/sections/specific.txt]: d15fc750822137058677b28f9e710cdb9cadb425

File: SpywareFilter/sections/specific.txt (hunk 1 above)
Comment (piquark6046) [reply SpywareFilter/sections/specific.txt]: d15fc750822137058677b28f9e710cdb9cadb425

**Review by Alex-302 — APPROVED** [review]

**Comment by Alex-302** [comment]
thanks

**Pull request merged**